Frontend/package.json

```diff
@@ -7,11 +7,19 @@
   "author": "GraphQLBase",
   "license": "MIT",
   "scripts": {
-    "up": "webpack --mode development",
-    "dev": "node ./dist/main.js",
-    "lint": "eslint src/**",
+    "build": "webpack --mode development",
+    "server": "node ./dist/server.js",
+    "up": "yarn build",
+    "lint": "eslint src/**/*.*",
     "test": "jest",
-    "build:production": "webpack --mode production"
+    "build:production": "webpack --mode production",
+    "dev": "run-p dev:nodemon dev:browsersync dev:watch",
+    "dev:nodemon": "nodemon --watch dist ./dist/server.js",
```

Comment:
this command automatically restarts the server when changes are made. We use `nodemon`, instead of `chokidar`, because it properly handles server crashes and restarting services on the same port.